tools/tester.rb

```diff
@@ -103,14 +213,24 @@ def print_results()
   def run_test(optional_vm_args, mutex)
     temp_out = Tempfile.new("dora-test-runner")
     cmdline = "#{binary} #{vm_args} #{optional_vm_args} #{test_file} #{args}"
-    stdout, stderr, status = Open3.capture3(cmdline)
-    result = check_test_run_result(stdout, stderr, status)
+    if timeout != nil
```

Comment:
Let's enforce a default timeout for every test (e.g. 60 seconds - not sure whether this makes sense).

Reply:
I set the default to 60 seconds. Hopefully, this time the actions do not fail.